README.md

```diff
@@ -294,7 +294,7 @@ updateNavigationBar = ({
 -   `expandedTitle`: updates NavigationBar expandedTitle. If the value is an
     empty string, the expanded navigation bar will not be shown. Only available
     in native app versions >= 11.8
--   `showBackButton`: shows or hides back icon in NavigationBar
+-   `showBackButton`: shows or hides back icon in NavigationBar. On Android it does not apply in the following cases: when the navigation mode is STAY or INTERNAL with MODAL presentation. In these cases back navigation will be performed with the native button.
```

Comment:
WDYT -> "On android, in case window does not support showing a back button (currently internal with modal presentation, which shows an X) this flag is ignored, as in these cases back navigation is always performed with the native back button."

The STAY you see in the code is because if a new window is launched with STAY mode (because it was opened as a deeplink for example) it does not have a previous mode from where it can "stay", so internal-modal presentation is assumed.

Reply:
Done!